src/components/PokeCard/PokeCard.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import FavoriteBorderIcon from "@material-ui/icons/FavoriteBorder";
+
+import { ListItem, Description } from "./styles";
+import { formatPokeId } from "../../utils/utils";
+
+const PokeCard = ({ image, id }) => {
+  return (
+    <ListItem>
+      <div>
+        <img src={image} alt="" />
+      </div>
+      <Description>
+        <span>{formatPokeId({id})}</span>
+        <button>
+          <FavoriteBorderIcon fontSize="large" />
+        </button>
+      </Description>
+    </ListItem>
+  );
+};
+
+export default PokeCard;
+
+/*
+              
+              
+              */
+
+/*<ListItem key={key}>
```

Comment:
Nao deixe codigo comentado aleatoriamente, se nao for para explicar algo, e melhor deletar